projects/batfish/src/main/antlr4/org/batfish/grammar/cisco/Cisco_interface.g4

```diff
@@ -681,8 +681,6 @@ if_null_block
       | VLT_PEER_LAG
       | VMTRACER
       | VPC
-      | VRRP
-      | VRRP_GROUP
```

Comment:
Where is VRRP_GROUP handled now?

Reply:
Could not find it in any configs we have. Not sure why it's there, so I removed it.